bin/blin.p6

```diff
@@ -379,6 +382,27 @@ sub save-overview {
 save-overview;
 
 
+note '🥞🥞 Saving the markdown output';
+sub save-markdown {
+  $save-lock.protect: {
+    my $markdown-output = '';
+
+    for @sorted-modules {
+      my $bisected = '';
+
+      next if (my $result = .done ?? .done.result !! Unknown) == OK;
+
+      $bisected = ", Bisected: `{ .bisected.substr(0, 8) }`"
+        if $result == Fail;
+      $markdown-output ~= "* [x] { .name } – { $result }{ $bisected }";
```

Comment:
It should be `[ ]` instead of `[x]` so that the ToDo item is not preselected.

Reply:
Can you turn these into links? I believe prepending `https://modules.raku.org/dist/` is going to work. There's some sort of escaping we have to do, but we can figure it out later.

Reply:
You mean [ these? ](http://test.com/) -- Will need to look into escaping in markdown.

Reply:
Yeah. Generally I don't expect a lot of garbage in module names, and this file will be pasted into a ticket semi-manually, so don't worry about escaping too much. But there are [these subs in Whateverable::Bits that work half of the time](https://github.com/Raku/whateverable/blob/8d160b40e763ce6f0103f6396cee69116977bb14/lib/Whateverable/Bits.pm6#L51-L59).